views/layout.pug

```diff
@@ -9,6 +9,15 @@ html(lang='en')
     link(rel='stylesheet' href='/lib/animate/animate.css')
     link(rel='stylesheet' href='/lib/bootstrap/css/bootstrap.min.css')
     link(rel='stylesheet' href='/css/quizzard.css')
+    link(rel='stylesheet' href='/lib/materialize/css/materialize.min.css')
+    link(rel='stylesheet' href='/css/navbar.css')
```

Comment:
I wonder if putting navbar.css before icons.css would cause issues with the icons? could you please check that the icons show up properly in the navbar?

Reply:
no issue yet

Reply:
interesting